Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -158,11 +163,13 @@ public void Roslyn_Settings_Analyzer_AssemblyExists()
             BuildAssertions.AssertExpectedPropertyValue(result.ProjectStateAfterBuild, "SonarQubeRunRoslynCodeAnalysis", "true");
 
             // Check the analyzer properties are set as expected
-            AssertExpectedAnalyzersExists(result,
-                "analyzer1.dll",
-                "c:\\myfolder\\analyzer2.dll",
-                "sonarLint.dll.xxx",
-                GetDummySonarLintFilePath());
+            List<string> expectedAnalyzers = new List<string>(GetSonarLintAnalyzerFilePaths());
+            expectedAnalyzers.Add("analyzer1.dll");
+            expectedAnalyzers.Add("c:\\myfolder\\analyzer2.dll");
+            expectedAnalyzers.Add("sonarLint.dll.xxx");
+            expectedAnalyzers.Add("c:\\sonarLint\\my.dll");
```

Comment:
c:\sonarlint\my.dll is a new case: it shouldn't be matched because "sonarlint" isn't in the file name.